src/store/modules/my_profile.js

```diff
@@ -33,6 +36,9 @@ export default {
     },
     setRelayData ({ commit }, relayData) {
       commit('setRelayData', relayData)
+    },
```

Comment:
I'm not sure where these changes keep coming from.  I think I had the same thing happen on my branch.  This should be removed from this PR though, then we can merge.

Reply:
done